packages/grafana-ui/src/components/BigValue/PercentChange.tsx

```diff
@@ -22,5 +22,5 @@ export const PercentChange = ({ percentChange, styles }: Props) => {
 };
 
 export const percentChangeString = (percentChange: number) => {
-  return percentChange?.toLocaleString(undefined, { style: 'percent', maximumSignificantDigits: 3 }) ?? '';
+  return (percentChange / 100).toLocaleString(undefined, { style: 'percent', maximumSignificantDigits: 3 }) ?? '';
```

Comment:
hm should we maybe investigate why `percentChange` value changed recently? It seems on latest main it is being returned as `100` and `1` on v11.1.x - this could mean that other places we use diffPercent could be broken due to whatever recent change modified this behavior

On v11.1.x
<img width="1504" alt="v11 1 x" src="https://github.com/user-attachments/assets/533be8a8-4e9f-42df-859b-e6fae4ce32e6">

On main branch
<img width="1469" alt="main" src="https://github.com/user-attachments/assets/afe0586e-3aba-461a-8553-ebc177d48c85">

Code console.log
<img width="797" alt="code" src="https://github.com/user-attachments/assets/920c3574-bd4e-448a-9224-d5a4eeaede87">


Using this test dashboard
[Stat % change test-1722917087264.json](https://github.com/user-attachments/files/16505061/Stat.change.test-1722917087264.json)